flower-search/get_data.sh

```diff
@@ -2,5 +2,5 @@
 TEST_WORKDIR=/tmp/jina/flower/
 mkdir -p ${TEST_WORKDIR}
 cd ${TEST_WORKDIR}
-curl http://www.robots.ox.ac.uk/~vgg/data/flowers/17/17flowers.tgz --output 17flowers.tgz
+curl https://www.robots.ox.ac.uk/~vgg/data/flowers/17/17flowers.tgz --output 17flowers.tgz
```

Comment:
For me the old one still works. But I agree `https` is preferable when available

Reply:
For me the old one works, but it downloads a redirect page, then `tar` command fails. Does it really download the correct archive for you?

Alternatively one could add -L flag to curl

Reply:
Oh I see. I only tested it in the browser.